### PR TITLE
Add the Basic Terminology tips about the Managing Inventory

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds the Basic Terminology tips about the Managing Inventory

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no 

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://github.com/magento/devdocs/pull/8970/files

## Links to Magento source code or PRs

_OPTIONAL. If this pull request references a file in a Magento codebase repository or a code PR, add it here._

- https://github.com/magento/inventory/wiki

### CODE REVIEW

File: src/catalog/inventory-management.md [hunk 1]
```diff
@@ -32,6 +32,25 @@ sections:
...
 Magento Inventory Management gives you the tools to manage your product inventory. Merchants with a single store to multiple warehouses, stores, pickup locations, drop shippers, and more can use these features to maintain quantities for sales and handle shipments to complete orders. You can track your inventory quantities, provide accurate salable stock amounts to customers for all of your websites, and ship according to recommendations based on distance or priority. You can also configure your preferred product configurations globally (for all stores and products), per source, and per product. These features grow with your business, allowing you to work from a single warehouse or complex shipping network with a few additional settings.
 
+Inventory Management features include:
+
+- Different configurations for merchants whose inventory originates from a single source and from multiple sources.
+- Stocks for tracking available aggregated quantities through assigned sources
+- Concurrent checkout protection
+- Shipment matching algorithms
+
+## Basic Terminology
```
Comment (leslietilling) [inline src/catalog/inventory-management.md:42]:
```suggestion
## Basic terminology
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for the new content, but this should be included in catalog/inventory-learn-more.html instead.

**Comment by EvheniiaKolkhiieva** [comment]
@leslietilling , thank you for reviewing 😊
I have implemented your suggestions, could you please approve?
Thank you!

File: src/catalog/inventory-management.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/inventory-management.md]: Done ✅

File: src/catalog/inventory-management.md [hunk 2]
```diff
@@ -32,6 +32,25 @@ sections:
 
 Magento Inventory Management gives you the tools to manage your product inventory. Merchants with a single store to multiple warehouses, stores, pickup locations, drop shippers, and more can use these features to maintain quantities for sales and handle shipments to complete orders. You can track your inventory quantities, provide accurate salable stock amounts to customers for all of your websites, and ship according to recommendations based on distance or priority. You can also configure your preferred product configurations globally (for all stores and products), per source, and per product. These features grow with your business, allowing you to work from a single warehouse or complex shipping network with a few additional settings.
 
+Inventory Management features include:
+
+- Different configurations for merchants whose inventory originates from a single source and from multiple sources.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/catalog/inventory-management.md]: Done ✅

File: src/catalog/inventory-management.md [hunk 3]
```diff
@@ -32,6 +32,25 @@ sections:
...
+- Stocks for tracking available aggregated quantities through assigned sources
+- Concurrent checkout protection
+- Shipment matching algorithms
+
+## Basic Terminology
+
+The following terms are important is understanding as you work with Inventory Management:
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/catalog/inventory-management.md]: Done ✅

**Comment by leslietilling** [comment]
This new content should be moved to the catalog/inventory-learn-more.html page.

**Comment by EvheniiaKolkhiieva** [comment]
My mistake @leslietilling , now all done, please approve 😊

File: src/catalog/inventory-management.md [hunk 4]
```diff
@@ -32,6 +32,25 @@ sections:
...
+It is important to understand the following terms as you work with Inventory Management:
+
+- **Sources** represent physical locations that store and ship available products. These locations can include warehouses, brick-and-mortar stores, distribution centers, and drop shippers. (Any location can be designated as a source for virtual products.)
+
+- **Stocks** map a sales channel (currently limited to websites) to source locations and on-hand inventory. A stock can map to multiple sales channels, but a sales channel can be assigned to only one stock.
+
+- **Aggregate Salable Quantity** is the total virtual inventory that can be sold through a sales channel. The amount is calculated across all sources assigned to a stock.
+
+- **Reservations** track deductions from the salable quantity as customers add products to carts and complete checkout. When an order ships, the reservation clears and deducts the shipped amounts from specific source inventory quantities.
+
```
Comment (leslietilling) [inline src/catalog/inventory-management.md:53]:
```suggestion

```

File: src/catalog/inventory-management.md [hunk 5]
```diff
@@ -31,7 +31,6 @@ sections:
 ---
 
 Magento Inventory Management gives you the tools to manage your product inventory. Merchants with a single store to multiple warehouses, stores, pickup locations, drop shippers, and more can use these features to maintain quantities for sales and handle shipments to complete orders. You can track your inventory quantities, provide accurate salable stock amounts to customers for all of your websites, and ship according to recommendations based on distance or priority. You can also configure your preferred product configurations globally (for all stores and products), per source, and per product. These features grow with your business, allowing you to work from a single warehouse or complex shipping network with a few additional settings.
-
 {:.bs-callout-info}
```
Comment (leslietilling) [inline src/catalog/inventory-management.md:34]:
```suggestion

{:.bs-callout-info}
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**